tools/make/kube.mk

```diff
@@ -49,7 +49,6 @@ kube-generate: $(tools/controller-gen) ## Generate code containing DeepCopy, Dee
 kube-test: manifests generate $(tools/setup-envtest) ## Run Kubernetes provider tests.
 	@$(LOG_TARGET)
```

Comment:
can you also update L51 to include `celvalidation` tag

Reply:
done